### PR TITLE
change group to normal groups and add the security boundary with a re…

### PR DESCRIPTION
**Pull request by Timsto**
…stricted management AU

### CODE REVIEW

File: Deploy-Policies.ps1 [hunk 1]
```diff
@@ -121,57 +121,145 @@ Import-Module -Name Microsoft.Graph.Beta.Groups
...
+    }
+}
+#endregion
+
+#region function for Add New Admin Assignment for RMAU
+function Add-AFAdminAssignmenttoRMAU {
+    [CmdletBinding()]
+    param (
+        $RMAUID,
+        $RoleAssignmentID = 'db004403-d48a-49c3-a098-088c45470d56',
```
Comment (xRDHS) [inline Deploy-Policies.ps1:195]: What is this being hardcoded to, is it an off the shelve role? If so which one? @Timsto?

**Review by xRDHS — COMMENTED** [review]
This is erroring on a fresh deployment, What's the name of the role?

File: Deploy-Policies.ps1 (hunk 1 above)
Comment (Timsto) [reply Deploy-Policies.ps1:195]: in multiplte tenant is was an identityfier for the Group Administrator.  so there is a Group Admin Scope on the RMAU that the CA Admin can manage the group

Comment (xRDHS) [reply Deploy-Policies.ps1:195]: Thats good, That makes sense, cos I had to manually add myself as a group admin, thought I missed a step in the documentation. I'll do a PR.

**Pull request merged**